src/redux/wallets/reducer.ts

```diff
@@ -26,10 +26,12 @@ import { RESET_ALL_DATA, EXTENSION_UPDATE_STATE } from '../app/actions';
 const intialState: IWalletsState = {};
 
 const newBalance = (oldBalance: any, action: any) => ({
-    value: action.data.balance ? action.data.balance : oldBalance?.value,
+    value: action.data.balance ? action.data.balance.total : oldBalance?.value,
```

Comment:
`value` should be `balance.available`

Reply:
I added available 2 lines after this. so right now we have: available, total and value. Value should still be available ? 
available: action.data.balance ? action.data.balance.available : oldBalance?.available,

Reply:
balance.value is used in many places in our app, and in most of the cases that value represents the available amount from user's account. Here you set the total amount on `value` which will lead to misleading information.

Reply:
If we change the `value` we must handle state migration as well
My suggestion for now: keep `value` as available and add `total` to use it where we need
@krisboit